tests/unit/system/classes/AutoDatasourceTest.php

```diff
@@ -7,7 +7,7 @@
 
 class CmsThemeTemplateFixture extends Model
 {
-    protected $fillable = ['*'];
+    protected $guarded = [];
```

Comment:
What was the reason for this change?

Reply:
`protected $fillable = ['*']` is actually incorrect. What happened was in our unit tests, when the seeders run on set up, they set all Eloquent models to be unguarded, so this was never picked up. However, Laravel must've improved their guarding, because the unit test that tests fillable attributes was failing. I've set the guarding for that fixture to use the correct method for making a single model unguarded.

Reply:
Perfect, thanks for the explanation!